datasets.py

```diff
@@ -15,9 +15,11 @@
 import audio
 import vad
 import transcripts
+import random
 
 class AudioTextDataset(torch.utils.data.Dataset):
-	def __init__(self, data_paths, labels, sample_rate, frontend = None, speakers = None, waveform_transform_debug_dir = None, min_duration = None, max_duration = None, mono = True, segmented = False, time_padding_multiple = 1, audio_backend = 'sox'):
+	def __init__(self, data_paths, labels, sample_rate, frontend = None, speakers = None, waveform_transform_debug_dir = None, min_duration = None, max_duration = None, mono = True, segmented = False, time_padding_multiple = 1, audio_backend = 'sox', exclude=[], load_first_n_sec=None):
+		self.load_first_n_sec = load_first_n_sec
```

Comment:
Let's instead introduce `max_duration_filter = True` and `max_ref_len` / `max_ref_len_filter = True`

Reply:
Then you'd need to apply `max_ref_len_filter` and `max_ref_len` as the last step:
1) filter out transcripts longer than `max_ref_len` if `max_ref_len_filter is True`
2) take only `max_ref_len` characters for those that we were kept in

Reply:
I'll handle this